variables.tf

```
@@ -36,13 +36,18 @@ variable "resource_group_name" {
 
 variable "auth_settings" {
   type = object({
-    enabled                       = bool
-    issuer                        = string
-    token_store_enabled           = bool
-    unauthenticated_client_action = string
-    active_directory = object({
-      client_id         = string
-      allowed_audiences = list(string)
+    enabled                = bool
+    default_provider       = string
+    require_authentication = bool
+    unauthenticated_action = string
+    active_directory_v2 = object({
```

Remove v2 suffix for compatibility

```suggestion
    active_directory = object({
```